src/sessions/SessionManager.ts

```diff
@@ -176,7 +176,11 @@ export class SessionManager {
     private createOrRenewSessionCookie(session: Session, expires: Date) {
         if (btoa) {
             storeCookie(
-                this.sessionCookieName(),
+                getCookieName(
+                    this.config.cookieAttributes.unique,
+                    SESSION_COOKIE_NAME,
+                    this.appMonitorDetails.id!
+                ),
```

Comment:
suggestion: Set the cookie name once in the constructor.
```
this.sessionCookieName = this.config.cookieAttributes.unique
            ? `${SESSION_COOKIE_NAME}_${this.appMonitorDetails.id}`
            : SESSION_COOKIE_NAME;
```
Then we can re-use the value.
```suggestion
                this.sessionCookieName,
```